### ISSUE TITLE
Smartlock panel accessibility update

### ISSUE THREAD

**Issue by megelliott** [google/WebFundamentals#8245]
**Page Affected:** Smartlock integration in web

**What needs to be done?** Currently the element has no visible focus indicator when smartlock has the focus. If a keyboard user is trying to log in to a site, they are unable to select from multiple accounts as tabbing does not alert them to where they are on the page (in the Smartlock panel) or what is selected (their account).

https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-focus-visible.html

![NYT_049](https://user-images.githubusercontent.com/47230223/67299916-3e222e80-f4bb-11e9-8d73-78334ec2fb12.png)
